src/main/java/com/customerApi/api/controllers/CustomergetController.java

```diff
@@ -0,0 +1,42 @@
+package com.customerApi.api.controllers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.customerApi.api.services.CustomerFetchService;
+
+
+
+@RestController
+public class CustomergetController {
+	
+	private static final Logger logger = LoggerFactory.getLogger(CustomergetController.class); 
+	
+	
+	@Autowired
+	private CustomerFetchService customerFetchService;
+	
+	@GetMapping("/getCustomer")
+	@ResponseBody
+	public String getCustomer()
+	{
+		logger.info("Sending request to service layer");
+		String data = customerFetchService.getCustomersData();		
+		return data;		
+		
+	}
+	@GetMapping("getCustomer/{id}")
+	@ResponseBody
+	public String getCustomerById(@PathVariable("id") int id)
+	{
+		logger.info("Redirect to service layer"+id);
```

Comment:
The proper way of printing log is below, its an example
_For example, if you want to print 1 value"_
logger.info("Customer Name {}, sign-in in", customer.getName());

_For example, if you want to print 2 values or multiple"_
logger.info("Customer First Name {}, last name {}, sign-in in", customer.getFirstName(), customer.getLastName());